linux/system/autoupdates.sls

```diff
@@ -0,0 +1,24 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.enabled %}
+
+{%- if system.autoupdates is defined %}
```

Comment:
This was because if `autoupdates.enabled = False`, I want to actually disable Unattended-Upgrade, so the state should still be processed